### PR TITLE
PBMC-21: Early model work

### PR DESCRIPTION
**Pull request by Chrisctr**

### CODE REVIEW

File: .idea/libraries/sbt__com_google_guava_listenablefuture_9999_0_empty_to_avoid_conflict_with_guava_jar.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <library name="sbt: com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava:jar">
     <CLASSES>
       <root url="jar://$USER_HOME$/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/google/guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar!/" />
+      <root url="jar://$USER_HOME$/AppData/Local/Coursier/cache/v1/https/repo1.maven.org/maven2/com/google/guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar!/" />
```
Comment (CarolineS-QA) [inline .idea/libraries/sbt__com_google_guava_listenablefuture_9999_0_empty_to_avoid_conflict_with_guava_jar.xml:5]: What dis? (where's this file at?)

Comment (Chrisctr) [reply .idea/libraries/sbt__com_google_guava_listenablefuture_9999_0_empty_to_avoid_conflict_with_guava_jar.xml:5]: No u

Comment (Femarleycode) [reply .idea/libraries/sbt__com_google_guava_listenablefuture_9999_0_empty_to_avoid_conflict_with_guava_jar.xml:5]: no u

**Review by CarolineS-QA — CHANGES_REQUESTED** [review]
Remove the .idea file. Please _/ \\_

Ohterwise LGTM :P

**Comment by Femarleycode** [comment]
LGTM:)

**Pull request merged**